plugins/doc_fragments/ldap.py

```diff
@@ -24,6 +24,11 @@ class ModuleDocFragment(object):
       - The password to use with I(bind_dn).
     type: str
     default: ''
+  ca_cert_file:
+    description:
+      - Set the path to PEM file with CA certs
+    type: str
```

Comment:
Shouldn't this be
```suggestion
    type: path
```
?

Reply:
Yes, `path` is more sensible. I wasn't aware that there is a `type='path'` tbh. Tanks!